config/custom.cfg

```diff
@@ -0,0 +1,17 @@
+[buildout]
+auto-checkout = Euphorie
+
+[settings]
+# main-package = your.euphorie.flavor
+saconfig-uri = sqlite:///app/var/quoira.sql
+
+[instance]
+# remove the zcml declaration if you configure a main-package
+# which is not Euphorie itself (the default)
+zcml =
+    euphorie.deployment-meta
+    euphorie.deployment
+    euphorie.deployment-overrides
+
+[versions]
+Euphorie =
```

Comment:
The provided pin in base.cfg is not available on the configured pypi servers.

Reply:
I think you are doing something wrong:

See:

1. https://github.com/euphorie/euphorie-project/blob/6fdc6387772e66eee42a63e9f751df791bef71a7/config/base.cfg#L122
2. https://pypi.org/project/Euphorie/18.0.0/